src/DatabaseInstance.php

```diff
@@ -374,6 +374,14 @@ public function rawSearchOptions()
             'joinparams'       => ['jointype' => 'child']
         ];
 
+        $tab[] = [
+            'id'                 => '6',
+            'table'              => DatabaseInstance::getTable(),
+            'field'              => 'version',
+            'name'               => __('Version'),
```

Comment:
You'll have an i18n issue on that one, it already exists with a plural form.

Reply:
```suggestion
            'name'               => _n('Version', 'Versions', 1),
```

Reply:
fixed